Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -258,11 +258,11 @@ static void EncodeToRamUsingShader(GLuint srcTexture,
 		glReadPixels(0, 0, (GLsizei)dstWidth, (GLsizei)dstHeight, GL_BGRA, GL_UNSIGNED_BYTE, nullptr);
 		u8* pbo = (u8*)glMapBufferRange(GL_PIXEL_PACK_BUFFER, 0, dstSize, GL_MAP_READ_BIT);
 
-		for (int i = 0; i < readLoops; i++)
+		for (int i = 0; i < dstHeight; i++)
```

Comment:
I'm pretty sure this will break strided efb copys (not the YUV XFB ones). There the dstStride applies per block line, not per pixel line. Feel free to fork this function into its two callers ;)